Source/Core/DolphinQt/GBAWidget.cpp

```diff
@@ -281,6 +298,10 @@ void GBAWidget::contextMenuEvent(QContextMenuEvent* event)
   unload_action->setEnabled(CanControlCore() && !m_game_title.empty());
   connect(unload_action, &QAction::triggered, this, &GBAWidget::UnloadROM);
 
+  auto* card_action = new QAction(tr("&Load e-Reader Card"), menu);
+  card_action->setEnabled(CanControlCore() && !m_game_title.empty());
```

Comment:
```suggestion
  card_action->setEnabled(CanControlCore());
```

Reply:
Don't you need to load the e-Reader ROM first?  Or does that happen automatically?

Reply:
e-Reader itself has to be loaded in order to scan cards.  It's like a normal game in how it is loaded in mGBA.

Reply:
Alright, then I think it makes sense to still require a non-empty game title if you can't load a card when the e-Reader itself is loaded.  (It'd be better to check if it's the e-Reader itself that's loaded, but I'm not sure how to do that.)